src/pages/patientView/clinicalInformation/PatientViewPageStore.ts

```diff
@@ -158,6 +168,16 @@ import {
     getMutationSubType,
     getSimplifiedMutationType,
 } from 'shared/lib/oql/AccessorsForOqlFilter';
+import {
+    CLINICAL_ATTRIBUTE_ID_ENUM,
+    MSI_H_THRESHOLD,
+    TMB_H_THRESHOLD,
+} from 'shared/constants';
+import {
+    MSI_H,
+    OTHER_BIOMARKER_HUGO_SYMBOL,
+    TMB_H,
+} from 'react-mutation-mapper/src/component/oncokb/constants';
```

Comment:
(See the previous related comment)
```suggestion
} from 'react-mutation-mapper';
```